code/edgeidentity/src/androidTest/java/com/adobe/marketing/mobile/edge/identity/util/ADBCountDownLatch.java

```diff
@@ -9,12 +9,15 @@
   governing permissions and limitations under the License.
 */
 
-package com.adobe.marketing.mobile.edge.identity;
```

Comment:
Do we want to expose the utils? I am just trying to understand the need for new package.